tests/test_fetch.py

```diff
@@ -0,0 +1,402 @@
+import pytest
+from typing import List
+from operator import itemgetter
+import itertools
+import numpy as np
+import decimal
+import pandas
+import warnings
+from . import schema
+import datajoint as dj
+import os
+import logging
+import io
+
+
+@pytest.fixture
+def lang():
+    yield schema.Language()
+
+
+@pytest.fixture
+def languages(lang) -> List:
+    og_contents = lang.contents
+    languages = og_contents.copy()
```

Comment:
I had to copy, store, and restore the `contents` of this table because the `contents` would change between tests. This caused some tests such as `test_fetch1_step1` to fail if other tests (that sort `contents`) such as `test_order_by` were run before it.